tests/test_aligner.yml

```diff
@@ -19,15 +19,25 @@
     - path: results/reference/bwamem2/genome.fasta.ann
     - path: results/reference/bwamem2/genome.fasta.bwt.2bit.64
     - path: results/reference/bwamem2/genome.fasta.pac
+    - path: results/reference/dbsnp/dbsnp_146.hg38.vcf.gz.tbi
+    - path: results/reference/fai/genome.fasta.fai
+    - path: results/reference/gatk4/genome.dict
+    - path: results/reference/intervals/chr22_1-40001.bed
+    - path: results/reference/intervals/genome.bed
+    - path: results/reference/known_indels/mills_and_1000G.indels.vcf.gz.tbi
+    - path: results/reference/target/chr22_1-40001.bed.gz
     - path: results/reports/fastqc/test-test_L1
+    - path: results/reports/markduplicates/test/test.md.metrics
```

Comment:
@maxulysse all the `.md` were changed to `.mapped` in my last PR. Were the prefixes changed again?

Reply:
no, only the .md produced by --skip markduplicates should have been changed from `markduplicates/test/test.md*` to `mapped/test/test.sorted*`

Reply:
Nevermind, it was the other way round. Sorry!